#274 Added script to output makefile-dependencies for Fortran modules.

hiker opened the pull request:
Fixes #274 - adds the example of dependency creator for makefiles.

arporter commented on example/create_dependencies.py:
```diff
@@ -0,0 +1,130 @@
+# Author: Joerg Henrichs, Bureau of Meteorology
+
+'''This file contains an fparser script that parses Fortran files
+and output the dependencies between these files suiteable for a Makefile.
```
"suitable"

hiker replied:
Done.

arporter commented on example/create_dependencies.py:
```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env python
```
This won't work on a system that only has 'python3' but I'm not sure what we do about that.
Maybe just add a README (which we should have in the examples directory anyway) and mention this limitation.

hiker replied:
Indeed. I've added it to the README as known issue.

arporter reviewed (changes requested):
Looks good - thanks Joerg. I think the only thing missing is a (short) README.md to explain what the script does. If you'd be so kind as to add a few lines for the `fparser_f2008.py` script that would be great.

Separately (possibly a new issue), we should really be testing these examples and currently we don't.

hiker commented:
> Looks good - thanks Joerg. I think the only thing missing is a (short) README.md to explain what the script does. If you'd be so kind as to add a few lines for the `fparser_f2008.py` script that would be great.

Done - hope I got it right. I've also added a known-issues and TODO list - maybe someone gets motivated (and I might do it if I need it 😁  ).

> 
> Separately (possibly a new issue), we should really be testing these examples and currently we don't.

Agreed, I've opened #279 ... actually, I just added a very simple makefile that executes the examples (same way as in PSyclone - i.e. it's executed, but we don't test if the output is as expected). But I did not integrate it into travis, since I assume we are moving to github CI anyway. I'll mention this in #279.

arporter reviewed (approved):
All good now. Thanks for adding the Makefile and README.
